Dockerfile

```diff
@@ -6,32 +6,41 @@ ARG BUILDNUM=""
 # Build Quai in a stock Go builder container
 FROM golang:1.20-alpine as builder
 RUN apk add --no-cache gcc musl-dev linux-headers git
-
 ADD . /go-quai
-
 WORKDIR /go-quai
-
 RUN env GO111MODULE=on go run build/ci.go install ./cmd/go-quai
 
 # Stage 2
 FROM golang:1.20-alpine
 
-EXPOSE 8546 8547 30303 30303/udp
-EXPOSE 8578 8579 30304 30304/udp
-EXPOSE 8580 8581 30305 30305/udp
-EXPOSE 8582 8583 30306 30306/udp
-EXPOSE 8610 8611 30307 30307/udp
-EXPOSE 8542 8643 30308 30308/udp
-EXPOSE 8674 8675 30309 30309/udp
-EXPOSE 8512 8613 30310 30310/udp
-EXPOSE 8544 8645 30311 30311/udp
-EXPOSE 8576 8677 30312 30312/udp
-EXPOSE 8614 8615 30313 30313/udp
-EXPOSE 8646 8647 30314 30314/udp
-EXPOSE 8678 8679 30315 30315/udp
+# Prime
+EXPOSE 10001 10002 10000 10000/udp
+
+# Region
+EXPOSE 10004 10005 10003 10003/udp
```

Comment:
if we are taking the effort to change the PORT numbers I recommend we make it more user friendly.

For Ex:

11000 -> Prime
10100 -> Region1
10200 -> Region2
10011 -> Zone11

Something like this, just think about it, if we are putting in effort, it would be nice if we could type port numbers in just by knowing the Name of the chain

Reply:
We might need to have a meeting about specific port numbers in that case. You can't do 100XY for zones because each zone and region need to have 3 ports available to them (HTTP, WS, TCP/UDP) so there would be collisions. The ports that I have reserved right now are a little confusing to read, but they make the scripting logic very easy

Reply:
I think regardless of the ports that you choose, it's going to be impossible to type the port numbers in just by knowing the name of the chain because you'll need to look up HTTP vs WS vs TCP/UDP

Reply:
I mean its possible right, just need a nice way to do it that makes sense, some form of encoding